content/docs/07-reference/ai-sdk-ui/03-use-assistant.mdx

```diff
@@ -128,6 +128,18 @@ This works in conjunction with [`AssistantResponse`](./assistant-response) in th
       description:
         'The current status of the assistant. This can be used to show a loading indicator.',
     },
+    {
+      name: 'append',
+      type: '(message: Message | CreateMessage, chatRequestOptions: { options: { headers, body } }) => Promise<string | undefined>',
+      description:
+        "Append a user message to the current thread. This triggers the API call to fetch the assistant's response.",
```

Comment:
looks like we use "Function to..." for `useChat`, should we update description here to be consistent?